README.md

```diff
@@ -43,7 +43,9 @@ Here are some commands you can run:
 | Command                     | Description |
 |-----------------------------|-------------|
 | yarn dev:amo                | Start the dev server and proxy (amo) |
+| yarn dev:amo:dev            | Start the dev server and proxy (amo) using data from the dev server (https://addons-dev.allizom.org/) |
```

Comment:
I thought about this, but adding more ENV vars would then require the config to check for them and change things based on it. I'm okay with that approach but it's a bit less obvious/easy-to-document-in-the-README. It's also more code as here we're just piggy-backing on existing configs–I think even here with the extra CSP code/tests it's less work.